### ISSUE TITLE
Plugin not working after `vuepress build`

### ISSUE THREAD

**Issue by thekiwi** [eFrane/vuepress-plugin-mermaidjs#40]
Hi! Your plugin works brilliantly when I run `vuepress dev docs`, however when I run `vuepress build docs` to prepare for deployment, the static site that is built no longer shows the diagrams. Instead, the spinner is shown. 

I can't see any errors in the console or any other reason for it not working. 

Here is my list of devDependencies:
```JSON
  "devDependencies": {
    "@vuepress/plugin-pwa": "1.7.1",
    "vuepress": "1.7.1",
    "vuepress-plugin-mermaidjs": "1.8.0"
  },
```

Any ideas what I'm doing wrong? Or where I can look to find out more information on why it's not working?

Thanks!

**Comment by thekiwi** [comment]
For anyone else facing a similar issue, I figured out what was wrong in my situation. 

I was actually running `vuepress build docs --debug`, **not** `vuepress build docs`. 

At first, this seems like an irrelevant difference, but I found a [related Vuepress issue](https://github.com/vuejs/vuepress/issues/1944#issuecomment-578543958) that describes this flag causing issues in other plugins. 

Removing this flag and re-running the build has sorted the diagrams out now. As such, I'll close this issue.
